src/gitlab.ts

```diff
@@ -37,6 +37,7 @@ const BUILTIN_ACCESS_LEVEL_ANONYMOUS = ['$anonymous', '$all'];
 
 // Level to apply on 'allow_access' calls when a package definition does not define one
 const DEFAULT_ALLOW_ACCESS_LEVEL = ['$all'];
+const DEFAULT_ALLOW_PUBLISH_LEVEL = ['$owned-group'];
```

Comment:
should be `$owner`

Reply:
If I were to do it the way you suggest, I think the default value should be `$maintainer` since it currently is the default value in the code. (https://github.com/bufferoverflow/verdaccio-gitlab/blob/2fa6944cc3f43c31f32127a61bb4e09b22504461/src/gitlab.ts#L68)

Reply:
yep, this makes sense